src/sdk-py/src/zk_binary.rs

```diff
@@ -0,0 +1,55 @@
+use darkfi::zkas::decoder;
+use pyo3::prelude::*;
+
+#[pyclass]
+pub struct ZkBinary(pub(crate) decoder::ZkBinary);
+
+/// There is no need for constants, as bindings for ec_mul_short and ec_mul_base
+/// don't actually take the constants.
+/// The constants are hardcoded on the Rust side.
+#[pymethods]
+impl ZkBinary {
+    #[staticmethod]
+    fn decode(bytes: Vec<u8>) -> Self {
+        let bincode = decoder::ZkBinary::decode(bytes.as_slice()).unwrap();
```

Comment:
Zkrunner uses `ZkBinary::decode`. Wdyt? Do you see more things from `zkas -e` we could use?